app/api/models/domain/MtdSourceEnum.scala

```diff
@@ -25,7 +25,7 @@ sealed trait MtdSourceEnum {
 
 object MtdSourceEnum {
 
-  case object hmrcHeld extends MtdSourceEnum {
+  case object `hmrc-held` extends MtdSourceEnum {
     override def toDesViewString: String = "HMRC-HELD"
```

Comment:
Is this meant to be "HMRC-HELD" or "HMRC HELD"?

Reply:
Actually looks ok, as the DES spec requires "HMRC-HELD" for the "view" query param (but "HMRC HELD" everywhere else), so the code's correct here.